vue/src/client/services/ApiService.ts

```diff
@@ -19,7 +19,6 @@ import { EvaluationImageResults } from "../../types";
 
 export interface QueryArguments {
   performer?: string[];
-  groundtruth?: boolean;
```

Comment:
I'm guessing this was removed because it isn't being used.  I might have added that before as a way to eventually only show the groundtruth but never implemented it in `views/model_run.py`